charts/incubator/proxmox-backup-server/values.yaml

```diff
@@ -0,0 +1,42 @@
+image:
+  repository: tccr.io/truecharts/proxmox-backup-server
+  tag: v2.2-7@sha256:dcbaee6378f32c5062518ea39a5e8e1260c3a3c609bcb32bd2dd406b7a9ef0d0
+  pullPolicy: IfNotPresent
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsUser: 0
+    runAsGroup: 0
+  pod:
+    fsGroup: 34
+
+service:
+  main:
+    ports:
+      main:
+        targetPort: 8007
+        protocol: https
```

Comment:
Since this is https, you probably need to setup the container probes to be https too

See example here
https://github.com/truecharts/charts/pull/11242/files